year-pages/2019-conferences-list.md

```diff
@@ -2,6 +2,6 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
-| [SACON 2019](https://www.sacon.io/) | 15-16 Feb | Banglore |  India’s 1st Security Architecture Conference | No
+| [SACON 2019](https://www.sacon.io/) | 15-16 Feb | Bangalore |  India’s 1st Security Architecture Conference | No
```

Comment:
@vikaskyadav thanks for catching that! Since we are correcting this anyway, can you change this to "Bengaluru"?